qualang_tools/utils/README.md

```diff
@@ -0,0 +1,53 @@
+# QUA Utils
+
+QUA utils is a collection of small Python or QUA functions and classes which make QUA programs clearer and easier.
+
+It is meant to be updated regularly so feel free to share any comments and suggestions following these
+ [instructions](https://github.com/qua-platform/py-qua-tools#support-and-contribution).

```

Comment:
Better to link directly here:
https://github.com/qua-platform/py-qua-tools/blob/main/CONTRIBUTING.md

Reply:
Done